packages/api-types/src/index.ts

```diff
@@ -1,3 +1,5 @@
+export { bookKeys } from './book-keys';
```

Comment:
Where should we put little bits of code/data that we need to share between the frontend and backend, to avoid duplicate code? Should we have a new nx package? I'm not sure how to set that up.

Reply:
Can we put this in the data folder next to the morphology script? Then the morphology script can use it as well.

Reply:
I need to remember to do this tomorrow morning.

Reply:
Done